[2024-12-02T14:02:47.282+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-02T14:02:47.325+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_hive_pipeline.consume_from_kafka manual__2024-12-02T14:00:26.702204+00:00 [queued]>
[2024-12-02T14:02:47.355+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_hive_pipeline.consume_from_kafka manual__2024-12-02T14:00:26.702204+00:00 [queued]>
[2024-12-02T14:02:47.356+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-02T14:02:47.427+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): consume_from_kafka> on 2024-12-02 14:00:26.702204+00:00
[2024-12-02T14:02:47.447+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_hive_pipeline', 'consume_from_kafka', 'manual__2024-12-02T14:00:26.702204+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/gestion_airflow.py', '--cfg-path', '/tmp/tmp_h_9oji4']
[2024-12-02T14:02:47.459+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1455) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-02T14:02:47.464+0000] {standard_task_runner.py:105} INFO - Job 141: Subtask consume_from_kafka
[2024-12-02T14:02:47.465+0000] {standard_task_runner.py:72} INFO - Started process 1469 to run task
[2024-12-02T14:02:47.687+0000] {task_command.py:467} INFO - Running <TaskInstance: kafka_hive_pipeline.consume_from_kafka manual__2024-12-02T14:00:26.702204+00:00 [running]> on host 45770b1a429e
[2024-12-02T14:02:47.984+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_hive_pipeline' AIRFLOW_CTX_TASK_ID='consume_from_kafka' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T14:00:26.702204+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-02T14:00:26.702204+00:00'
[2024-12-02T14:02:47.989+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-02T14:02:48.962+0000] {hive.py:475} INFO - USE `default`
[2024-12-02T14:02:49.675+0000] {hive.py:475} INFO - 
    CREATE TABLE IF NOT EXISTS sales (
        id STRING,
        invoice_id STRING,
        branch STRING,
        city STRING,
        customer_type STRING,
        gender STRING,
        product_line STRING,
        unit_price FLOAT,
        quantity INT,
        tax_5_percent FLOAT,
        total FLOAT,
        `date` STRING,
        time STRING,
        payment STRING,
        cogs FLOAT,
        gross_margin_percentage FLOAT,
        gross_income FLOAT,
        rating FLOAT
    ) STORED AS ORC
    
[2024-12-02T14:02:49.824+0000] {kafka_to_hive_with_redis.py:70} INFO - Subscribed to topic 'topic-hive'
[2024-12-02T14:14:41.156+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-12-02T14:14:41.158+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-02T14:14:41.159+0000] {process_utils.py:132} INFO - Sending 15 to group 1469. PIDs of all processes in the group: [1469]
[2024-12-02T14:14:41.160+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1469
[2024-12-02T14:14:44.935+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-02T14:14:45.193+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-02T14:14:45.232+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1469, status='terminated', exitcode=0, started='14:02:46') (1469) terminated with exit code 0
